site/docs/master/install-overview.md

```diff
@@ -34,6 +34,8 @@ not using another storage platform that offers an S3-compatible object storage A
 Second, if you need to back up persistent volume data, you must select a volume backup solution. [volume snapshot providers][0] contains information on the supported options. For example, if you use [Portworx][6] for persistent storage, you can install their Velero plugin to get native Portworx snapshots as part of your Velero backups. If there is no native snapshot plugin available for your storage platform, you can use Velero's [restic integration][4], which provides a
 platform-agnostic backup solution for volume data.
 
+Third, if you need to install Velero without a default backup storage location (without specifying `--bucket` or `--provider`), the `--no-default-backup-location` flag is required for confirmation.
```

Comment:
I'm not sure it makes sense to put this in the "On-premises" section - maybe under "Customize configuration" instead?

Reply:
Sure, I think that makes sense. I will try to move it into the 'Customize configuration'.